app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -29,40 +32,52 @@
 
 @Service
 public class ExperimentPageContentService {
+    public static final String INFERRED_CELL_TYPE_ONTOLOGY_LABELS_FROM_DB = "inferred cell type - ontology labels";
+    public static final String INFERRED_CELL_TYPE_AUTHORS_LABELS_FROM_DB = "Inferred cell type - authors labels";
```

Comment:
I suggest to make these 2 variable private.

```suggestion
    private static final String INFERRED_CELL_TYPE_ONTOLOGY_LABELS_FROM_DB = "inferred cell type - ontology labels";
    private static final String INFERRED_CELL_TYPE_AUTHORS_LABELS_FROM_DB = "Inferred cell type - authors labels";
```

Thanks